homeassistant/components/netatmo/webhook.py

```diff
@@ -62,21 +62,31 @@ def async_evaluate_event(hass, event_data):
             person_event_data[ATTR_IS_KNOWN] = person.get(ATTR_IS_KNOWN)
             person_event_data[ATTR_FACE_URL] = person.get(ATTR_FACE_URL)
 
-            async_send_event(hass, event_type, person_event_data)
+            await async_send_event(hass, event_type, person_event_data)
 
     else:
-        _LOGGER.debug("%s: %s", event_type, event_data)
-        async_send_event(hass, event_type, event_data)
+        await async_send_event(hass, event_type, event_data)
 
 
-@callback
-def async_send_event(hass, event_type, data):
+async def async_send_event(hass, event_type, data):
```

Comment:
We don't await inside this function so we didn't need to convert it to a coroutine function.

Reply:
Thanks for the hint. I think there is even more unnecessary async keywords. Did some clean up here: https://github.com/home-assistant/core/pull/47037